### DISCUSSION TITLE
High Memory Utilization on Single Node, other_ets Consumption

### DISCUSSION THREAD

**Discussion by iamhritik** [rabbitmq/rabbitmq-server#11108] [Other] [answered]
Hey everyone,

We've encountered an unexpected issue with one of our nodes experiencing a significant spike in memory utilization. Upon investigation, we've found that the majority of the memory is being consumed by **other_ets**
Any insights or advice on resolution are appreciated.

<img width="500" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/47936485/0f8fe1cd-35cc-41ab-a481-8b0324f43454">

**Environment:**
RabbitMQ: 3.11.5
Erlang: 25.3.2.10

**queue type**: quorum

<img width="1294" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/47936485/c6259fc5-15a2-424b-ba33-047180b94bd0">

**rabbitmq conf**
```
listeners.tcp.1 = 0.0.0.0:5672
mqtt.listeners.tcp.1 = 0.0.0.0:1443
mqtt.tcp_listen_options.backlog = 9182
mqtt.tcp_listen_options.recbuf  = 191072
mqtt.tcp_listen_options.sndbuf  = 191072
mqtt.tcp_listen_options.keepalive = true
mqtt.tcp_listen_options.nodelay   = true
mqtt.tcp_listen_options.exit_on_close = true
mqtt.tcp_listen_options.send_timeout  = 120
mqtt.allow_anonymous  = false
mqtt.default_user     = guest
mqtt.default_pass     = guest
mqtt.vhost            = /
mqtt.exchange         = amq.topic
mqtt.subscription_ttl = 86400000
management.listener.port = 15672
management.listener.ip   = 0.0.0.0
log.file.level = info
vm_memory_high_watermark.relative = 0.6
handshake_timeout = 20000
tcp_listen_options.backlog = 9182
tcp_listen_options.recbuf  = 191072
tcp_listen_options.sndbuf  = 191072
tcp_listen_options.keepalive = true

```

**Comment by michaelklishin** [comment, marked as answer]
@iamhritik please stop repeatedly filing identical issues or you will be banned from the RabbitMQ org.

Your previous issue was converted into a discussion and answered in https://github.com/rabbitmq/rabbitmq-server/discussions/11117. The suggestion is to [upgrade](https://www.rabbitmq.com/docs/upgrade) to 3.13.1 with [a completely reworked MQTT plugin](https://www.rabbitmq.com/blog/2023/03/21/native-mqtt). 3.11.x is [out of community support](https://www.rabbitmq.com/release-information).

**Reply by iamhritik** [reply]
okk cool
